diff --git a/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec b/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
index 1234567..abcdefg 100644
--- a/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
+++ b/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
@@ -27,7 +27,6 @@
         "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
     }
     s.dependency "React-Codegen"
-    s.dependency "RCT-Folly"
     s.dependency "RCTRequired"
     s.dependency "RCTTypeSafety"
     s.dependency "ReactCommon/turbomodule/core"
